Doc/README.rst

```diff
@@ -130,7 +130,7 @@ Contributing
 ============
 
 Bugs in the content should be reported to the
-`Python bug tracker <https://bugs.python.org>`_.
+`Python bug tracker <https://github.com/python/cpython/issues>`_.
```

Comment:
Let's be consistent with [devguide](https://devguide.python.org/tracker/) and call it the "issue tracker":

```suggestion
`Python issue tracker <https://github.com/python/cpython/issues>`_.
```

Reply:
Superseded by later comment